src/MusicTheory/Music.elm

```diff
@@ -0,0 +1,325 @@
+eighthNoteTriplet m1 m2 m3 =
+    modify (Tuplet (Tuplet.triplet Tuplet.Eighth)) (line [ m1 Duration.eighthNote, m2 Duration.eighthNote, m3 Duration.eighthNote ])
+
+
+
+-- NOTE CONSTRUCTORS
+
+
+cFlat : Octave -> Duration -> Music Pitch
+cFlat o duration =
+    note duration (Pitch.pitch C Pitch.flat o)
+
+
+cNatural : Octave -> Duration -> Music Pitch
+cNatural o duration =
+    note duration (Pitch.pitch C Pitch.natural o)
+
+
+cSharp : Octave -> Duration -> Music Pitch
+cSharp o duration =
+    note duration (Pitch.pitch C Pitch.sharp o)
+
+
+dFlat : Octave -> Duration -> Music Pitch
+dFlat o duration =
+    note duration (Pitch.pitch D Pitch.flat o)
+
+
+dNatural : Octave -> Duration -> Music Pitch
+dNatural o duration =
+    note duration (Pitch.pitch D Pitch.natural o)
+
+
+dSharp : Octave -> Duration -> Music Pitch
+dSharp o duration =
+    note duration (Pitch.pitch D Pitch.sharp o)
+
+
+eFlat : Octave -> Duration -> Music Pitch
+eFlat o duration =
+    note duration (Pitch.pitch E Pitch.flat o)
+
+
+eNatural : Octave -> Duration -> Music Pitch
+eNatural o duration =
+    note duration (Pitch.pitch E Pitch.natural o)
+
+
+eSharp : Octave -> Duration -> Music Pitch
+eSharp o duration =
+    note duration (Pitch.pitch E Pitch.sharp o)
+
+
+fFlat : Octave -> Duration -> Music Pitch
+fFlat o duration =
+    note duration (Pitch.pitch F Pitch.flat o)
+
+
+fNatural : Octave -> Duration -> Music Pitch
+fNatural o duration =
+    note duration (Pitch.pitch F Pitch.natural o)
+
+
+fSharp : Octave -> Duration -> Music Pitch
+fSharp o duration =
+    note duration (Pitch.pitch F Pitch.sharp o)
+
+
+gFlat : Octave -> Duration -> Music Pitch
+gFlat o duration =
+    note duration (Pitch.pitch G Pitch.flat o)
+
+
+gNatural : Octave -> Duration -> Music Pitch
+gNatural o duration =
+    note duration (Pitch.pitch G Pitch.natural o)
+
+
+gSharp : Octave -> Duration -> Music Pitch
+gSharp o duration =
+    note duration (Pitch.pitch G Pitch.sharp o)
+
+
+aFlat : Octave -> Duration -> Music Pitch
+aFlat o duration =
+    note duration (Pitch.pitch A Pitch.flat o)
+
+
+aNatural : Octave -> Duration -> Music Pitch
+aNatural o duration =
+    note duration (Pitch.pitch A Pitch.natural o)
+
+
+aSharp : Octave -> Duration -> Music Pitch
+aSharp o duration =
+    note duration (Pitch.pitch A Pitch.sharp o)
+
+
+bFlat : Octave -> Duration -> Music Pitch
+bFlat o duration =
+    note duration (Pitch.pitch B Pitch.flat o)
+
+
+bNatural : Octave -> Duration -> Music Pitch
+bNatural o duration =
+    note duration (Pitch.pitch B Pitch.natural o)
+
+
+bSharp : Octave -> Duration -> Music Pitch
+bSharp o duration =
+    note duration (Pitch.pitch B Pitch.sharp o)
+
+
+
+-- FUNCTIONS
+
+
+line : List (Music a) -> Music a
+line ms =
+    case ms of
+        [] ->
+            rest Duration.zero
+
+        [ m ] ->
+            m
+
+        h :: t ->
+            List.foldl (\m1 m2 -> seq m2 m1) h t
+
+
+chord : List (Duration -> Music a) -> Duration -> Music a
```

Comment:
I changed the way to create chords. The duration has to be passed to the constructor so that it's not possible (very easily) to create chord with notes that have different durations.

It is still possible to create parallel voices, tough with the `par` constructor.

Here: https://github.com/elm-music/music-theory/pull/35/files#diff-4345091e9177eabbdf790cdff1e2e15fR21 you can see the usage of it.